iijpta/ngx_http_secure_token_iijpta.c

```diff
@@ -25,6 +25,10 @@
 #define CRC32_SIZE    4
 #define EXPIRY_SIZE   8
 #define PATH_LIMIT    1024
+#define SEMICOLON_SPC 2
```

Comment:
Instead of these 3 macros, I'd prefer - `#define COOKIE_ATTR_SIZE (sizeof("; Expires=Thu, 31-Dec-2019 23:59:59 GMT; Max-Age=") - 1 + NGX_TIME_T_LEN)` (notice I used 4 digits for the year, there are cases where ngx_http_cookie_time does that)